htdocs/accountancy/admin/card.php

```diff
@@ -135,25 +138,28 @@
 		// To manage zero or not at the end of the accounting account
 		if($conf->global->ACCOUNTING_MANAGE_ZERO == 1)
 		{
-			$account_number = GETPOST('account_number');
+			$account_number = GETPOST('account_number','int');
```

Comment:
account_number is a string. checking if int introduce serious troubles in accountancy module
Same for some other fields